docs/development/module/tests.md

```diff
@@ -60,7 +60,14 @@ fn fake_test_example() {
 }
 ```
 
-Custom implementations of [Externalities](https://crates.parity.io/substrate_externalities/index.html) allow developers to construct runtime environments that provide access to features of the outer node. Another example of this can be found in [`substrate-offchain`](https://crates.parity.io/substrate_offchain/), which maintains its own [Externalities](https://crates.parity.io/substrate_offchain/testing/index.html) implementation. [Implementing configurable externalities](https://substrate.dev/recipes/testing/externalities.htm) is covered in more depth in the recipes.
+Custom implementations of
+[Externalities](https://crates.parity.io/substrate_externalities/index.html) allow developers to
+construct runtime environments that provide access to features of the outer node. Another example of
+this can be found in [`substrate-offchain`](https://crates.parity.io/substrate_offchain/), which
+maintains its own [Externalities](https://crates.parity.io/substrate_offchain/testing/index.html)
+implementation. [Implementing configurable
+externalities](https://substrate.dev/recipes/testing/externalities.html) is covered in more depth in
+the recipes.
```

Comment:
Note to reviewers: The only substantial change in this blob is the second link. it now correctly ends in `.html`

Reply:
I am quickly testing links:

- https://crates.parity.io/substrate_externalities/index.html
- https://crates.parity.io/substrate_offchain/
- https://crates.parity.io/substrate_offchain/testing/index.html

all point to 404

Btw, what is the context about having https://crates.parity.io/ and https://substrate.dev/rustdocs/ together?

Reply:
I see you are only fixing links pointing to recipe now.